src/automation.md

```diff
@@ -108,12 +118,14 @@ jobs:
         TOKEN: ${{secrets.MY_SUPER_SECRET_STORED_KEY}}
 ```
 
-If this section stimulated your curiosity and you want to find out more, click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action).
+Still don't know how this .yml works? click [here](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes)
+
+If this section stimulated your curiosity and you want to find out more, click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action)
 
-Still don't know how this .yml works? click [here](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes).
+### Questions
```

Comment:
Please add Questions for this section.

Reply:
- What are the benefits from CD/CI?

Would be one I have in mind. Without questions I would not add the Headline. We could start with this one and iteratively add more.